src/Metable.php

```diff
@@ -202,7 +202,7 @@ protected function getDefaultMetaValue($key)
      */
     public function getAllMeta(): \Illuminate\Support\Collection
     {
-        return collect($this->defaultMetaValues)->merge(
+        return collect($this->defaultMetaValues ?? [])->merge(
```

Comment:
perhaps it would be clearer to do a `property_exists()` check similar to `hasDefaultMeta()`? You could extract that logic to a method such `getAllDefaultMeta()`

Reply:
I 've turned it to method. Please, if you still insist on applying `property_exists`, ping me to changes